scripts/install_dev_tools.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat, Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+# Match the .venv created by the Makefile
+DEFAULT_VENV="${SCRIPT_DIR}/../.venv"
```

Comment:
The `SCRIPT_DIR` thing gets the real path in case it's a symbolic link, right?

- Could we pass in the venv from the makefile, so we're not duplicating the logic for it?
- If not, does it matter if it's a symlink or not?
- If it does matter, can we comment this line out with what it does? I only figured it out by trying it.

Reply:
- It gets the PATH from which it's being invoked. If we want to have exact symlink location then $(pwd) would be used, but I don't see any use case for that.

```
mkdir /tmp/test
cd ~
ln -s test /tmp/test
./test/install_dev_tools
# SCRIPT_DIR --->    /home/user/test/install_dev_tools
/tmp/test/install_dev_tools
# SCRIPT_DIR --->    /tmp/test/
```

Added comment inside the script.


As for the second question:
venv is passed from the Makefile, this one is only to have it included in the script in case someone wants to run it outside of Makefile.

```
cli_dev_tools:
	./scripts/install_dev_tools.sh -v $(PELORUS_VENV)
```

Reply:
Oh right, I didn't think about how it was trying to go upwards. Good call.